### PR TITLE
SQL

### PR DESCRIPTION
**Pull request by Ravi1004**
Created registration page, linked it with sql database, created admin page and retrieved data from the table.

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -1,29 +1,66 @@
...
 
 app = Flask(__name__)
 
 # Check for environment variable
 if not os.getenv("DATABASE_URL"):
     raise RuntimeError("DATABASE_URL is not set")
 
+
 # Configure session to use filesystem
+app.config["SQLALCHEMY_DATABASE_URI"] = "postgres://yainmriqptbdia:ef9eb65b36e26462fab4fa3190ea7982f65a5b450e7be9789b4ac2f507b8f634@ec2-54-81-37-115.compute-1.amazonaws.com:5432/d226orf1s2l1ac"
```
Comment (manasa133) [inline project1/application.py:28]: get the url from console, don't add it here as a part of security

File: project1/application.py [hunk 2]
```diff
@@ -1,29 +1,66 @@
...
+    Firstname = request.form.get("fname")
+    Lastname = request.form.get("lname")
+    Name = Firstname+" "+Lastname
+    username = request.form.get("uname")
+    password = request.form.get("pass")
+    credentials = User(username=username, password=password)
+    db.session.add(credentials)
+    db.session.commit()
+    # s = "Hello " + Firstname + " " + Lastname + ": Your Username is " + username
+    return render_template("registered.html", message=Name + ", account is sucessfully registered.")
```
Comment (manasa133) [inline project1/application.py:70]: indicate not successful also

File: project1/models.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+class User(db.Model):
+    _tablename_ = "users"
+    username = db.Column(db.String, primary_key=True)
+    password = db.Column(db.String, nullable=False)
+    user_created_on = db.Column(db.DateTime, nullable=False)
+
+    def _init_(self, username, password):
+        self.username = username
+        self.password = password
+        self.user_created_on = datetime.now()
```
Comment (manasa133) [inline project1/models.py:16]: its correct

File: project1/application.py (hunk 2 above)
Comment (Ravi1004) [reply project1/application.py:70]: In branch3 i made this change ma'am

File: project1/application.py (hunk 1 above)
Comment (Ravi1004) [reply project1/application.py:28]: In branch i changed this one

**Comment by manasa133** [comment]
Merge with master branch

**Pull request merged**